examples/FP8_QUANT/README.md

```diff
@@ -10,7 +10,7 @@ This is an example of mature FP8, which under the hood leverages some functional
 ## Requirements
 
 - [FMS Model Optimizer requirements](../../README.md#requirements)
-- Nvidia A100 family or higher
+- Nvidia H100 family or higher
```

Comment:
vllm can actually run FP8 in a special mode on A100s... i.e., cast FP8->FP16, hence run FP8 models without having real FP8 compute engine. it should give us the same results because upcasting will not change the "content" of the tensor, just put a smaller data in a bigger container.

Reply:
ok, rolled back